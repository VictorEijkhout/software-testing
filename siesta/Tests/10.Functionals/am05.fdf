%include ../base_graph.fdf
SystemLabel     am05

xc.functional   GGA
xc.authors      AM05

