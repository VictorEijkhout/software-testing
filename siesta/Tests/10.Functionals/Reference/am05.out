Siesta Version  : 5.0.0-alpha-123-gfb8fd7d02
Architecture    : ----
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch;-O3 -march=native
PP flags        : ----
Libraries       : ----
Parallelisations: none
GEMM3M support
NetCDF support
NetCDF-4 support
Lua support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/Tests/10.Functionals/work/am05
* Running in serial mode.
>> Start of run:  28-SEP-2023  15:38:09

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

reinit: Reading from standard input
reinit: Dumping input in INPUT_TMP.82850
************************** Dump of input data file ****************************
%include ../../basejob.fdf
SystemLabel     am05
xc.functional   GGA
xc.authors      AM05
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: Graphite
reinit: -----------------------------------------------------------------------
reinit: System Label: am05
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:    6 Label: C


---- Processing specs for species: C
Ground state valence configuration:   2s02  2p02
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/C.psf

---- Pseudopotential check for C 

Pseudized shells:
2s( 2.00) rc: 1.49
2p( 2.00) rc: 1.52
3d( 0.00) rc: 1.58
Valence configuration for ps generation: (assumed as above)
For C, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.
C pseudopotential only contains V_ls up to l=2 -- lmxkb reset.

<basis_specs>
===============================================================================
C                    Z=   6    Mass=  12.011        Charge= 0.17977+309
Lmxo=2 Lmxkb= 2    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=2
          i=1  nzeta=1  polorb=0  (2s)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=1  polorb=1  (2p)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=2  Nsemic=0  Cnfigmx=3
          i=1  nzeta=0  polorb=0  (3d)  (perturbative polarization orbital) (from 2p)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Filter: Requested cutoff:   50.000000 Ry
Using PAO.SplitTailNorm
atom: Called for C                     (Z =   6)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    4.00000

xc_check: Exchange-correlation functional:
xc_check: GGA AM05
xc_check: WARNING: Pseudopotential generated with LDA CA functional
V l=0 = -2*Zval/r beyond r=  1.4801
V l=1 = -2*Zval/r beyond r=  1.5087
V l=2 = -2*Zval/r beyond r=  1.5628
All V_l potentials equal beyond r=  1.5612
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.5628
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside     17.810 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     40.589 Ry
atom: Maximum radius for chlocal:    1.86354
atom: Maximum radius for r*vlocal+2*Zval:    1.70915

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.745634   el= -1.003143   Ekb=  5.184193   kbcos=  0.301074
GHOST: No ghost state for L =  1
   l= 1   rc=  1.745634   el= -0.395823   Ekb= -4.329594   kbcos= -0.369390
GHOST: No ghost state for L =  2
   l= 2   rc=  1.934160   el=  0.012726   Ekb= -1.015591   kbcos= -0.010473

KBgen: Total number of Kleinman-Bylander projectors:   9
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 2s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.003675 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    5.017032
                 energy =   -0.998867
                kinetic =    0.843730
    potential(screened) =   -1.842597
       potential(ionic) =   -5.419899
PAO Filter: Cutoff used:   24.50 Ry
PAO Filter: diagnostic kin energy tol:    0.003000 Ry
PAO Filter: Nominal cutoff (before filtering):   26.56 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 26  6

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 2p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.003675 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    6.179348
                 energy =   -0.391607
                kinetic =    2.420651
    potential(screened) =   -2.812258
       potential(ionic) =   -6.263879
PAO Filter: Cutoff used:   24.50 Ry
PAO Filter: diagnostic kin energy tol:    0.003000 Ry
PAO Filter: Nominal cutoff (before filtering):   41.39 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 30  8

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 2p
Split based on tail norm

   izeta = 1
                     rc =    6.179348
                 energy =    1.062107
                kinetic =    2.160791
    potential(screened) =   -1.098684
       potential(ionic) =   -3.881584
PAO Filter: diagnostic kin energy tol:    0.003000 Ry
PAO Filter: Nominal cutoff (before filtering):   44.32 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 30  7
atom: Total number of Sankey-type orbitals:  9

atm_pop: Valence configuration (for local Pseudopot. screening):
 2s( 2.00)                                                            
 2p( 2.00)                                                            
 3d( 0.00)                                                            
Vna: chval, zval:    4.00000   4.00000

Vna:  Cut-off radius for the neutral-atom potential:   6.179348
VNA Filter: diagnostic kinetic energy tol:    0.003000 Ry
VNA Filter: Nominal cutoff (before filtering):   21.82 Ry
VNA Filter: Cutoff used:    50.00 Ry
Filter:    Number of eigenfunctions  of the
           filtering kernel (total, used)= 38 12

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1    6 C                       # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
C                     2                    # Species label, number of l-shells
 n=2   0   1                         # n, l, Nzeta 
   5.017   
   1.000   
 n=2   1   1 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   6.179   
   1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file C.ion.nc
coor:   Atomic-coordinates input format  =     Fractional

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   0.00000  1        1
siesta:      0.00000   0.00000   6.33814  1        2
siesta:     -0.00000   2.68503   0.00000  1        3
siesta:      2.32531   1.34252   6.33814  1        4

siesta: System type = bulk      

initatomlists: Number of atoms, orbitals, and projectors:      4    36    36

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = none
redata: Number of spin components                   = 1
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        1
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = NO
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =    50.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =     1000
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   F
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   299.9863 K
redata: Fix the spin of the system                  =   F
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 2
mix.SCF:    Linear mixing weight                 =     0.250000
mix.SCF:    Mixing weight                        =     0.250000
mix.SCF:    SVD condition                        = 0.1000E-07
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.2500
  weight.linear 0.2500
  history 2
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:    16.000000
Total ionic charge:    16.000000
 Kpoints in:           25 . Kpoints trimmed:           25

siesta: k-grid: Number of k-points = 25
siesta: k-points from cutoff
siesta: k-cutoff (effective) =     6.152 Ang
siesta: k-point supercell and displacements
siesta: k-grid:     0     5     0      0.000
siesta: k-grid:     5     0     0      0.000
siesta: k-grid:     0     0     2      0.500

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000

superc: Internal auxiliary supercell:     9 x     9 x     3  =     243
superc: Number of atoms, orbitals, and projectors:    972   8748   8748


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

superc: Internal auxiliary supercell:     9 x     9 x     3  =     243
superc: Number of atoms, orbitals, and projectors:    972   8748   8748

outcell: Unit cell vectors (Ang):
        2.461000    0.000000    0.000000
       -1.230500    2.131289    0.000000
        0.000000    0.000000    6.708000

outcell: Cell vector modules (Ang)   :    2.461000    2.461000    6.708000
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000    120.0000
outcell: Cell volume (Ang**3)        :     35.1841
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=36 nrows=36 sparsity=56.5833 nnzs=73332, refcount: 7>
  <dData1D:(new from dSpData1D) n=73332, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=36 nrows=36 sparsity=56.5833 nnzs=73332, refcount: 8>
  <dData2D:DM n=73332 m=1, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      82     381

InitMesh: MESH = 10 x 10 x 30 = 3000
InitMesh: Mesh cutoff (required, used) =    50.000    55.279 Ry
New grid distribution [1]: sub = 2

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -244.843983
siesta: Eions   =       997.163120
siesta: Ena     =       203.289890
siesta: Ekin    =       406.597088
siesta: Enl     =       -42.011976
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =        -1.296886
siesta: DUscf   =         0.711043
siesta: DUext   =         0.000000
siesta: Ex      =      -161.643642
siesta: Ec      =       -22.222671
siesta: Exc     =      -183.866312
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -604.119057
siesta: Etot    =      -613.740273
siesta: FreeEng =      -613.740273

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1     -604.119057     -613.740273     -613.740273  1.371067 -5.264977  0.405752
timer: Routine,Calls,Time,% = IterSCF        1       0.108  23.00
   scf:    2     -613.757183     -613.748855     -613.748855  0.002901 -5.172740  0.273034
   scf:    3     -613.764395     -613.757196     -613.757196  0.004887 -5.945944  0.023622
   scf:    4     -613.757282     -613.757240     -613.757240  0.000474 -5.947449  0.016325
   scf:    5     -613.757325     -613.757286     -613.757286  0.000772 -5.951759  0.003930
   scf:    6     -613.757291     -613.757289     -613.757289  0.000211 -5.952178  0.001385
   scf:    7     -613.757290     -613.757289     -613.757289  0.000123 -5.952376  0.000200
   scf:    8     -613.757289     -613.757289     -613.757289  0.000011 -5.952382  0.000131

SCF Convergence by DM+H criterion
max |DM_out - DM_in|         :     0.0000114973
max |H_out - H_in|      (eV) :     0.0001309760
SCF cycle converged after 8 iterations

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      82     381

siesta: E_KS(eV) =             -613.7573

siesta: E_KS - E_eggbox =      -613.7573

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot    0.000000    0.000000    0.000000
----------------------------------------
   Max    0.008822
   Res    0.004210    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    0.008822    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):     -330.70     -330.57       71.76        0.00       -0.00        0.11
(Free)E + p*V (eV/cell)     -609.4420
Target enthalpy (eV/cell)     -613.7573

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -246.192535
siesta: Eions   =       997.163120
siesta: Ena     =       203.289890
siesta: Ekin    =       404.872073
siesta: Enl     =       -41.671749
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =        -0.180486
siesta: DUscf   =         0.649407
siesta: DUext   =         0.000000
siesta: Ex      =      -161.339307
siesta: Ec      =       -22.213998
siesta: Exc     =      -183.553304
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -613.757289
siesta: Etot    =      -613.757289
siesta: FreeEng =      -613.757289

siesta: Final energy (eV):
siesta:  Band Struct. =    -246.192535
siesta:       Kinetic =     404.872073
siesta:       Hartree =     160.279062
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:         Exch. =    -161.339307
siesta:         Corr. =     -22.213998
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =    -183.553304
siesta:  Ion-electron =    -731.414095
siesta:       Ion-ion =    -263.941026
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =    -613.757289
siesta:         Fermi =      -5.952382

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.206406    0.000067   -0.000000
siesta:     0.000067   -0.206328    0.000000
siesta:    -0.000000    0.000000    0.044792

siesta: Cell volume =         35.184138 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00133580          0.00133431  Ry/Bohr**3
siesta:           0.12264740          0.12251028  eV/Ang**3
siesta:         196.50279894        196.28310376  kBar
(Free)E+ p_basis*V_orbitals  =        -611.903283
(Free)Eharris+ p_basis*V_orbitals  =        -611.903283

cite: Please see "am05.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.0.0-alpha-123-gfb8fd7d02
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =       1.555
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       1.555       1.555   100.00
Setup                  1       0.062       0.062     3.97
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1       1.492       1.492    95.97
geom_init              1       0.299       0.299    19.25
state_init             1       0.066       0.066     4.27
hsparse                1       0.007       0.007     0.46
overlap                1       0.034       0.034     2.21
Setup_H0               1       0.188       0.188    12.12
naefs                  2       0.000       0.000     0.03
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.021       0.043     2.73
nlefsm                 2       0.115       0.230    14.80
DHSCF_Init             1       0.031       0.031     1.98
DHSCF1                 1       0.001       0.001     0.08
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.030       0.030     1.90
REMESH                 1       0.003       0.003     0.22
REORD                 28       0.000       0.000     0.01
PHION                  1       0.024       0.024     1.53
COMM_BSC              43       0.000       0.000     0.01
POISON                11       0.000       0.001     0.04
fft                   22       0.000       0.000     0.02
setup_H                9       0.045       0.403    25.94
DHSCF                 10       0.059       0.590    37.92
DHSCF3                10       0.044       0.444    28.57
rhoofd                10       0.023       0.235    15.11
XC                    10       0.002       0.022     1.39
GXC-CellXC            10       0.002       0.022     1.39
gridxc@cellXC         10       0.002       0.022     1.39
vmat                  10       0.019       0.187    12.01
IterSCF                8       0.110       0.878    56.49
compute_dm             8       0.064       0.511    32.89
diagon                 8       0.064       0.510    32.81
c-eigval             200       0.001       0.157    10.09
c-buildHS            200       0.001       0.141     9.06
cdiag                400       0.000       0.046     2.95
cdiag1               400       0.000       0.002     0.16
cdiag2               400       0.000       0.009     0.56
cdiag3               400       0.000       0.032     2.06
c-eigvec             200       0.001       0.170    10.94
cdiag4               200       0.000       0.001     0.04
c-buildD             200       0.001       0.182    11.73
MIXER                  7       0.000       0.003     0.18
PostSCF                1       0.314       0.314    20.21
DHSCF4                 1       0.145       0.145     9.35
dfscf                  1       0.143       0.143     9.19
overfsm                1       0.010       0.010     0.63
state_analysis         1       0.000       0.000     0.01
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.001       0.001     0.03
optical                1       0.000       0.000     0.00
  

>> End of run:  28-SEP-2023  15:38:10
Job completed
